website/docs/usage/training.mdx

```diff
@@ -575,6 +575,29 @@ now-updated model to the predicted docs.
 
 </Infobox>
 
+### Using rehearsing to address catastrophic forgetting {id="rehearse-components", tag="experimental", version="3.5.1"}
+
+Perform “rehearsal” updates to pre-trained components. Rehearsal updates teach the current component to make predictions similar to an initial model, to try to address the “catastrophic forgetting” problem. This feature is experimental.
+
+```ini {title="config.cfg (excerpt)"}
+[nlp]
+pipeline = ["sentencizer", "ner", "entity_linker"]
+
+[components.ner]
+source = "en_core_web_sm"
+
+[training]
+rehearse_components = ["ner"]
+```
+
+<Infobox variant="warning" title="Loss calculation" id="rehearse-components-loss">
+
+Be aware that the loss is calculated by the sum of both the `update` and `rehearse` function. 
+If both the loss and accuracy of the component increases over time, it can be caused due to the trained component making more different predictions that the inital model,
```

Comment:
Why would an increase in accuracy indicate catastrophic forgetting?

Reply:
**My understanding is that:**

If the accuracy of the model improves on a test data set, which has a similar domain as the training data, while the loss is also increasing, we can assume that the trained model is learning new data, but at the same time, the loss/difference between the teacher and student predictions is also getting higher, which might indicate that the model is forgetting.
I can be completely wrong here, but that's how I would interpret it 😇

Reply:
Shouldn't we than report `rehearse` and `update` losses separately actually?

Reply:
Yes, I think that would make more sense